UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Activities/DictionaryContainsKey.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Activities;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using UiPathTeam.Extensions.Activities.Properties;
+using UiPath.Shared.Activities;
+using UiPath.Shared.Activities.Localization;
+
+namespace UiPathTeam.Extensions.Activities
+{
+    [LocalizedDisplayName(nameof(Resources.DictionaryContainsKey_DisplayName))]
+    [LocalizedDescription(nameof(Resources.DictionaryContainsKey_Description))]
+    public class DictionaryContainsKey : ContinuableAsyncCodeActivity
+    {
+        #region Properties
+
+        /// <summary>
+        /// If set, continue executing the remaining activities even if the current activity has failed.
+        /// </summary>
+        [LocalizedCategory(nameof(Resources.Common_Category))]
+        [LocalizedDisplayName(nameof(Resources.ContinueOnError_DisplayName))]
+        [LocalizedDescription(nameof(Resources.ContinueOnError_Description))]
+        public override InArgument<bool> ContinueOnError { get; set; }
+
+        [LocalizedDisplayName(nameof(Resources.DictionaryContainsKey_In_dictionary_DisplayName))]
+        [LocalizedDescription(nameof(Resources.DictionaryContainsKey_In_dictionary_Description))]
+        [LocalizedCategory(nameof(Resources.Input_Category))]
+        public InArgument<Dictionary<object, object>> In_dictionary { get; set; }
+
+        [LocalizedDisplayName(nameof(Resources.DictionaryContainsKey_In_key_DisplayName))]
+        [LocalizedDescription(nameof(Resources.DictionaryContainsKey_In_key_Description))]
+        [LocalizedCategory(nameof(Resources.Input_Category))]
+        public InArgument<object> In_key { get; set; }
+
+        [LocalizedDisplayName(nameof(Resources.DictionaryContainsKey_Out_result_DisplayName))]
+        [LocalizedDescription(nameof(Resources.DictionaryContainsKey_Out_result_Description))]
+        [LocalizedCategory(nameof(Resources.Output_Category))]
+        public OutArgument<bool> Out_result { get; set; }
+
+        [LocalizedDisplayName(nameof(Resources.DictionaryContainsKey_Out_value_DisplayName))]
+        [LocalizedDescription(nameof(Resources.DictionaryContainsKey_Out_value_Description))]
+        [LocalizedCategory(nameof(Resources.Output_Category))]
+        public OutArgument<object> Out_value { get; set; }
+
+        #endregion
+
+
+        #region Constructors
+
+        public DictionaryContainsKey()
+        {
+        }
+
+        #endregion
+
+
+        #region Protected Methods
+
+        protected override void CacheMetadata(CodeActivityMetadata metadata)
+        {
+
+            base.CacheMetadata(metadata);
+        }
+
+        protected override async Task<Action<AsyncCodeActivityContext>> ExecuteAsync(AsyncCodeActivityContext context, CancellationToken cancellationToken)
+        {
+            // Inputs
+            var in_dictionary = In_dictionary.Get(context);
+            var in_key = In_key.Get(context);
+            var out_result = Out_result.Get(context);
```

Comment:
Out_result.Get(context) is always null; this assignment is unnecessary

Reply:
My code does not compile without this assignments. When I remove, i get errors below.

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0103	The name 'out_value' does not exist in the current context	UiPathTeam.Extensions.Activities (net461), UiPathTeam.Extensions.Activities (net6.0)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\Activities\DictionaryContainsKey.cs	78	Active
Error	CS0103	The name 'out_value' does not exist in the current context	UiPathTeam.Extensions.Activities (net461), UiPathTeam.Extensions.Activities (net6.0)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\Activities\DictionaryContainsKey.cs	88	Active
Error	CS0103	The name 'out_result' does not exist in the current context	UiPathTeam.Extensions.Activities (net461), UiPathTeam.Extensions.Activities (net6.0)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\Activities\DictionaryContainsKey.cs	77	Active
Error	CS0103	The name 'out_result' does not exist in the current context	UiPathTeam.Extensions.Activities (net461), UiPathTeam.Extensions.Activities (net6.0)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\Activities\DictionaryContainsKey.cs	82	Active
Error	CS0103	The name 'out_result' does not exist in the current context	UiPathTeam.Extensions.Activities (net461), UiPathTeam.Extensions.Activities (net6.0)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\Activities\DictionaryContainsKey.cs	87	Active
Error	CS0006	Metadata file 'C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\obj\Debug\net6.0\ref\UiPathTeam.Extensions.Activities.dll' could not be found	UiPathTeam.Extensions.Tests (net461)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions.Tests\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\bin\Debug\net461\UiPathTeam.Extensions.Activities.dll' could not be found	UiPathTeam.Extensions.Activities.Design (net461)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities.Design\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities\bin\Debug\net461\UiPathTeam.Extensions.Activities.dll' could not be found	UiPathTeam.Extensions.Tests (net461)	C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions.Tests\CSC	1	Active
Error		Could not find a part of the path 'C:\Users\michael.tomlin\source\repos\UiPathTAMTeam.Extensions.Activities\UiPathTeam.Extensions\UiPathTeam.Extensions.Activities.Design\obj\Debug\net6.0-windows7.0\UiPathTeam.Extensions.Activities.Design_0l5m1de5_wpftmp.GeneratedMSBuildEditorConfig.editorconfig'.	UiPathTeam.Extensions.Activities.Design	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\Roslyn\Microsoft.Managed.Core.targets	191

Reply:
you still need to define the variables, but they should not be assigned to values that are null
var out_result;

Reply:
Gotcha, thanks. I believe the other activities in the project also assign a value in the same I did mistakenly. Fixed.